pages/docs/workflows.md

```diff
@@ -5,14 +5,10 @@ next-section = "docs/cdn"
 prev-section = "index"
 +++
 
-# Workflow Guides
+# Generating Import Maps
 
-Some simple workflows for getting started with JSPM import map generation.
+The [Online Generator](#online-generator), [VSCode Extension](#vscode-extension), or the [JS Generator API](#js-generator-api) can be used to generate import maps for your project, with the Generator API being the recommended and most direct approach. There is also support for more advanced and experimental [Deno import maps](#deno-import-maps).
```

Comment:
Perhaps the sentence should start with the goal before the links?

```suggestion
Import maps can be generated for projects using the [Online Generator](#online-generator), [VSCode Extension](#vscode-extension), or the [JS Generator API](#js-generator-api), with the Generator API being the recommended and most direct approach. There is also support for more advanced and experimental [Deno import maps](#deno-import-maps).
```

Also typically we try to use passive voice - without using words like _you_ or _we_.